cli_tools/gce_vm_image_import/importer/disk_mutation_processor.go

```diff
@@ -0,0 +1,79 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	daisyCompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+	"google.golang.org/api/compute/v1"
+)
+
+type diskMutationProcessor struct {
+	args              ImportArguments
+	computeDiskClient daisyCompute.Client
+}
+
+func (d *diskMutationProcessor) process(pd persistentDisk) (persistentDisk, error) {
+	// If UEFI_COMPATIBLE is enforced in user input args (b.uefiCompatible),
+	// then it has been honored in inflation stage, so no need to recreate a new disk here.
+	if d.args.UefiCompatible {
+		return pd, nil
+	}
+
+	if !pd.isUEFIDetected {
+		return pd, nil
+	}
```

Comment:
minor: can be merged, but it's also ok to keep them separate for readability

Reply:
Yes I separate them so the comments can be read better